pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -871,4 +871,9 @@ self: super: {
     postInstall = "rm $out/bin/mkReadme && rmdir $out/bin";
   });
 
+  # These claim to have executables, but don't
```

Comment:
I don't believe that this is the right approach. Instead, that issue must be fixed in cabal2nix. The tool should simply not consider executables that have `buildable = false` when generating the appropriate Nix expression. I'll fix that ASAP and re-generate hackage-packages.nix. Anyhow, please don't include these changes in this PR.